### DISCUSSION TITLE
"error": "Transaction failed: insufficient funds for gas * price + value"

### DISCUSSION THREAD

**Discussion by Ferielle** [community/community#145104] [Programming Help]
### Select Topic Area

Question

### Body

Hello, I’m currently facing this bug:
"error": "Transaction failed: insufficient funds for gas * price + value"

I worked with Ganache, Truffle, PHP 8, and Laravel. I’m running out of time and encountering this error message. Could you please assist me?

"error": "Transaction failed: insufficient funds for gas * price + value"

PS: I’ve tried debugging by increasing the gas price and gas limit in Ganache, using the default settings, and manually increasing these values in the code. I’m attempting to send a small amount from a wallet to a contract created in my Ganache environment.

**Comment by officialzubairansari** [comment]
can you please share the model's code? so that we can see at least whats happening.

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @Ferielle!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

**Comment by jgschmitz** [comment]
@Ferielle  something like this? you didn't post any code so...... 
```
const transaction = {
  to: contractAddress,
  value: web3.utils.toWei("0.01", "ether"), // Sending 0.01 ETH
  gas: 300000, // Adjust as needed
  gasPrice: web3.utils.toWei("20", "gwei"), // Set a reasonable gas price
};
```

**Reply by LEOSOLAR8** [reply]
Thank you for your good response. Now I am developing the crypto trading platform. I wanna know much from you. Do you know about TradingView? If possible, I wanna be your friend. Warm regards.

**Comment by anasooo** [comment]
Check if your sending account has sufficient ETH in Ganache by using `web3.eth.getBalance(senderAddress)`. 
Then, set higher gas limits in your transaction: 
gas: `3000000`, 
gasPrice: `web3.utils.toWei('20', 'gwei')`. 
Make sure your transaction value + gas fees don't exceed your wallet balance
